SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+Due to frequent breaking changes from the HoYoverse/Cognosphere regarding game repair indexes, cache files, update zip files, etc. we can only support any latest Collapse version, be it either from Stable or Preview branch.

```

Comment:
Rephrase: Due to frequent breaking changes from HoYoverse/Cognosphere regarding update functionality (game repair indexes, cache files, compressed update packages, etc.) we can only officially guarantee support the latest Stable & Preview versions of Collapse.

Reply:
Applied